packages/wrapped-keys-lit-actions/src/lib/common/exportPrivateKey.js

```diff
@@ -13,31 +16,21 @@ const { removeSaltFromDecryptedKey } = require('../utils');
  */
 
 (async () => {
-  let decryptedPrivateKey;
   try {
-    decryptedPrivateKey = await Lit.Actions.decryptToSingleNode({
+    const decryptedPrivateKey = await getDecryptedKey({
       accessControlConditions,
       ciphertext,
       dataToEncryptHash,
-      chain: 'ethereum',
-      authSig: null,
     });
-  } catch (err) {
-    const errorMessage =
-      'Error: When decrypting to a single node- ' + err.message;
-    Lit.Actions.setResponse({ response: errorMessage });
-    return;
-  }
 
-  if (!decryptedPrivateKey) {
-    // Exit the nodes which don't have the decryptedData
-    return;
-  }
+    if (!decryptedPrivateKey) {
+      // Silently exit on nodes which didn't run the `decryptToSingleNode` code
+      return;
+    }
 
-  try {
     const privateKey = removeSaltFromDecryptedKey(decryptedPrivateKey);
     Lit.Actions.setResponse({ response: privateKey });
   } catch (err) {
-    Lit.Actions.setResponse({ response: err.message });
+    Lit.Actions.setResponse({ response: `Error: ${err.message}` });
```

Comment:
New structure ensures all errors thrown from the LA result in `Error:` prefix in the response.

Reply:
Since these are Lit Action errors should we prefix the error msg with "Lit Action Error" or the `err.message` displays that already?

Reply:
In order to get the error, it has to be parsed from a node response -- so the SDK throws a specific type of error indicating that it was an error running a request against a LIT node and the stack trace from the SDK indicates it comes from handling a node response